internal/dbutil/pgutil/schema.go

```diff
@@ -8,18 +8,46 @@ import (
 	"crypto/rand"
 	"database/sql"
 	"encoding/hex"
+	"io"
 	"net/url"
 	"strconv"
 	"strings"
 )
 
-// RandomString creates a random safe string
-func RandomString(n int) string {
+const (
+	randRetries = 2
+)
+
+// RandomStringFromReader creates a random safe string sourced from the specified reader.
+func RandomStringFromReader(n int, r io.Reader) string {
```

Comment:
There's a much easier fix for this, since this was used only for testing we can use `math/rand`, which doesn't fail. Then rename the method to `RandomTestSchema`.